libraries/fof/controller/controller.php

```diff
@@ -3347,56 +3347,15 @@ protected function _csrfProtection()
 				break;
 		}
 
-		$hasToken = false;
-		$session  = JFactory::getSession();
+		$token = new \Joomla\Cms\Session\CsrfToken(JFactory::getSession());
```

Comment:
Even if FOF is going to remove, I try to evolve without breaking stuff. That part in FOF was using JSessions token handling and would break when I switch to FW Session 2.0. So, if FOF gets removed anyway, it does not not hurt to keep it functioning.